Makefile

```diff
@@ -201,12 +201,29 @@ xgo:
 install-os-dependencies:
 	_assets/scripts/install_deps.sh
 
-setup: install-os-dependencies dep-install lint-install mock-install gen-install update-fleet-config ##@other Prepare project for first build
+setup: install-os-dependencies dep-install lint-install mock-install deploy-install gen-install update-fleet-config ##@other Prepare project for first build
 
 generate: ##@other Regenerate assets and other auto-generated stuff
 	go generate ./static ./static/migrations
 	$(shell cd ./services/shhext/chat && exec protoc --go_out=. ./*.proto)
 
+prepare-release:
```

Comment:
> I would not automate the bumping of VERSION (i.e. having the CI system do that), as I prefer this action to be explicit (we can of course provide a script to bump minor/major).

The problem here is that after a release we will keep the old version and if someone creates alpha, beta etc., it will refer as `already.released.version-beta` while it should be `upcoming.release-beta`.

I agree with the rest. Let's create `VERSION` file then and simplify the workflow.